content/javascript/concepts/arrays/terms/at/at.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.at()'
+Description: 'Return the element at a specified index in an array.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.at()`** method in JavaScript is used to return the element at a specified index in an array. The index can be either positive or negative, and negative indices count from the end of the array.
+
+## Syntax
+
+```js
+array.at(index);
+```
+
+- `index`:The element at the specified `index` is the one intended to be returned.
+
+## Example 1
+
```

Comment:
There should be an introductory sentence here such as:
The following code demonstrates a basic implementation of `.at()`: